docs/rules/prefer-explicit-local-imports.md

```diff
@@ -0,0 +1,27 @@
+# Prefer relative local imports extend to the file from where they are importing without relying on an index file. (prefer-explicit-local-imports)
```

Comment:
I think I might describe this more as "no ancestor directory import".

Reply:
Yeah, all makes sense. I like that name.